packages/features/eventtypes/components/AssignAllTeamMembers.tsx

```diff
@@ -9,12 +9,10 @@ const AssignAllTeamMembers = ({
   assignAllTeamMembers,
   setAssignAllTeamMembers,
   onActive,
-  onInactive,
 }: {
   assignAllTeamMembers: boolean;
   setAssignAllTeamMembers: Dispatch<SetStateAction<boolean>>;
   onActive: () => void;
-  onInactive?: () => void;
```

Comment:
Remove this because it isn't being used anywhere. Can add it later if use case arises